install_config/install/prerequisites.adoc

```diff
@@ -640,6 +640,18 @@ SELINUX=enforcing
 SELINUXTYPE=targeted
 ----
 
+[IMPORTANT]
```

Comment:
We may want to separate the note about the installer failing and multi tenancy because they really are two separate comments or topics.